source/features/more-dropdown.tsx

```diff
@@ -30,34 +30,38 @@ async function init(): Promise<void> {
 		createDropdown();
 	}
 
+	const insightsTab = select<HTMLAnchorElement>('[data-selected-links~="pulse"]')!;
+	const securityTab = select<HTMLAnchorElement>('[data-selected-links~="security"]')!;
+
 	select('.reponav-dropdown .dropdown-menu')!.append(
-		<a href={`/${repoUrl}/compare`} className="rgh-reponav-more dropdown-item" data-skip-pjax>
+		<a href={`/${repoUrl}/compare`} className="rgh-reponav-more dropdown-item">
 			{icons.darkCompare()} Compare
 		</a>,
 
 		isEnterprise() ? '' :
-			<a href={`/${repoUrl}/network/dependencies`} className="rgh-reponav-more dropdown-item rgh-dependency-graph" data-skip-pjax>
+			<a href={`/${repoUrl}/network/dependencies`} className="rgh-reponav-more dropdown-item">
 				{icons.dependency()} Dependencies
```

Comment:
I almost dropped this altogether because of the new "Used by" counter. For actual dependencies I have https://github.com/npmhub/npmhub